apps/emqx_bridge_kinesis/src/emqx_bridge_kinesis.erl

```diff
@@ -28,6 +34,37 @@ namespace() ->
 roots() ->
     [].
 
+fields(Field) when
+    Field == "get_connector";
+    Field == "put_connector";
+    Field == "post_connector"
+->
+    emqx_connector_schema:api_fields(
+        Field,
+        ?CONNECTOR_TYPE,
+        connector_config_fields()
```

Comment:
Missing the `resource_opts` configuration
<img width="647" alt="image" src="https://github.com/emqx/emqx/assets/13825269/628c54f2-b4db-446c-80e9-afd9ec2ad871">

e.g:
https://github.com/emqx/emqx/blob/dbe2baa4057aa0215501c003b0a56761bee07fac/apps/emqx_bridge_mongodb/src/emqx_bridge_mongodb.erl#L53-L56